markdown/events/2023/hackathon-march-2023/index.md

```diff
@@ -80,19 +80,19 @@ The documentation group will focus on all things documentation related, from pip
 
 #### Infrastructure
 
-The nf-core website and central tooling are always being improved. If you enjoy working with python or want to do some web development (mainly CSS and javascript (we are using astrojs+svelte)), join the infrastructure group!
+The nf-core website and central tooling are always being improved. If you enjoy working with python or want to do some web development (mainly CSS and javascript - we are using astrojs+svelte), join the infrastructure group!
 
 #### Pipelines
 
-The pipelines group will focus on any pipeline-related work. This could be a DSL2 conversion of a pipeline, a new or improved feature for a pipeline, or an entirely new pipeline (please let us register a new pipeline already before the hackathon, following [our guidelines](https://nf-co.re/docs/contributing/adding_pipelines#before-you-start)).
+The pipelines group will focus on any pipeline-related work. This could be a DSL2 conversion of a pipeline, a new or improved feature for a pipeline, or an entirely new pipeline (please let us register a new pipeline already before the hackathon, following [our guidelines](https://nf-co.re/docs/contributing/adding_pipelines#before-you-start)). These are typically lead by the main pipeline developers,  if they also attending the hackathon.
```

Comment:
```suggestion
The pipelines group will focus on any pipeline-related work. This could be a DSL2 conversion of a pipeline, a new or improved feature for a pipeline, or an entirely new pipeline (please let us register a new pipeline already before the hackathon, following [our guidelines](https://nf-co.re/docs/contributing/adding_pipelines#before-you-start)). These are typically lead by the main pipeline developers, if they are also attending the hackathon.
```